docs/upgrade/UPGRADE-unreleased.md

```diff
@@ -10,6 +10,81 @@ There you can find links to upgrade notes for other versions too.
 - *(low priority)* to add support of functional tests of Redis ([#846](https://github.com/shopsys/shopsys/pull/846))
     - download [`snc_redis.yml`](https://github.com/shopsys/project-base/blob/master/app/config/packages/test/snc_redis.yml) to `app/config/packages/test/snc_redis.yml`
     - download [`RedisFacadeTest.php`](https://github.com/shopsys/project-base/tree/master/tests/ShopBundle/Functional/Component/Redis/RedisFacadeTest.php) to your tests directory `tests/ShopBundle/Functional/Component/Redis/`
+- *(low priority)* to improve your deployment process and avoid possible Redis cache problems during deployment, include `build-version` into your builds ([#886](https://github.com/shopsys/shopsys/pull/886))
+    - to `app/AppKernel.php` into function `getConfig()` add
+        ```diff
+        private function getConfig()
+        {
+            // ...
+        +    if (file_exists(__DIR__ . '/config/parameters_version.yml')) {
+        +        $configs[] = __DIR__ . '/config/parameters_version.yml';
+        +    }
+
+            return $configs;
+        }
+        ```
+    - to `app/config/.gitignore` add
+        ```diff
+        + parameters_version.yml
+        ```
+    - configure redis cache clients in `app/config/packages/snc_redis.yml` to use `build-version` prefixes, eg.
+        ```diff
+        snc_redis:
+            clients:
+                bestselling_products:
+                # ...
+        -           prefix: '%env(REDIS_PREFIX)%bestselling_products_'
+        +           prefix: '%env(REDIS_PREFIX)%%build-version%bestselling_products_'
+        ```
+        **But be careful, don't add prefixes to the session client**
+
+        And add a `global` client (it is also without the `build-version` prefix)
+        ```diff
+        snc_redis:
+            clients:
+        +       global:
+        +           type: 'phpredis'
+        +           alias: 'global'
+        +           dsn: 'redis://%redis_host%'
+        +           options:
+        +               prefix: '%env(REDIS_PREFIX)%'
```

Comment:
should not be here also `%build-version%` ???

Reply:
It's used only in `RedisVersionFacade` and this facade has to be aware of all entries (to be able to delete the old one).